sigstore/verify/verifier.py

```diff
@@ -246,17 +250,32 @@ def verify(
 
         # 5) Verify the inclusion proof supplied by Rekor for this artifact.
         #
-        # We skip the inclusion proof only if explicitly requested.
-        if not materials._offline:
+        # The inclusion proof should always be present in the online case. In
+        # the offline case, if it is present, we verify it.
```

Comment:
I think it may have been optional because it doesn't provide any value over the SET when the client doesn't have any local state.

Reply:
Just to clarify: this means that transparency log entries in turn are mandatory within bundles, right? Since the inclusion proof appears within each of them 🙂

Reply:
Opened https://github.com/sigstore/protobuf-specs/pull/79.

Reply:
Ah, GitHub didn't show those last two comments for me...

In that case, does it make sense for the log entries to be mandatory (i.e., there should be at least one), but with optional interior inclusion proofs? If so, I can update the protobuf-specs PR.

Reply:
I think required for all seems reasonable. We're probably going to need a new version for future bundle updates around witnessing anyways as I bet they'll be breaking.

Reply:
edit: I'm also good with having inclusion proofs be optional because of the above, and basically tacklign this later on (maybe in a non breaking way)